src/module/event.js

```diff
@@ -0,0 +1,28 @@
+const checkboxChange = (toDoTask, checkbox, items, text) => {
+  toDoTask.completed = checkbox.checked;
+  localStorage.setItem("todoList", JSON.stringify(items));
+  if (checkbox.checked) {
+    text.style.textDecoration = "line-through";
+    // icon.style.display = "block";
+    // trashIcon.style.display = "none";
+  } else {
+    text.style.textDecoration = "none";
+    // trashIcon.style.display = "block";
+    // icon.style.display = "none";
+  }
+};
+
+export const clearCompletedTasks = (tasks, toDo) => {
+  tasks = tasks.filter((task) => !task.completed);
+  localStorage.setItem("todoList", JSON.stringify(tasks));
+  toDo.displayList();
+};
+
+// ...existing code...
+
+// const initializeClearButton = () => {
+//   const clearButton = document.getElementById("clear");
+//   clearButton.addEventListener("click", clearCompletedTasks);
+// };
```

Comment:
- [ ] Similarly, as mentioned above, remove all the commented code in order to make the codebase clean.